### PR TITLE
Change Profile version from 5.6.0 to 5.5.2 for Stack release

### PR DESCRIPTION
**Pull request by cormacpayne**
## Description

Changing the version of `AzureRM.Profile` for Stack release to be 5.5.2 to 5.6.0 to avoid conflict with the version of Profile being release for Ignite.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/1328/

File: tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.Commands.Profile.dll.json [hunk 1]
```diff
@@ -330,7 +330,7 @@
           "Type": {
             "Namespace": "Microsoft.Azure.Commands.Profile.Common",
```
Comment (bganapa) [inline tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.Commands.Profile.dll.json]: Whats the use of this file?

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/1330/

File: tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.Commands.Profile.dll.json (hunk 1 above)
Comment (markcowl) [reply tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.Commands.Profile.dll.json]: this is the record of the public interface, used by the breaking change tool

**Pull request merged**